src/JupyterLibrary/clients/jupyterlab/Shell.resource

```diff
@@ -61,9 +61,17 @@ Open With JupyterLab Menu
     END
 
 Maybe Accept A JupyterLab Prompt
-    [Documentation]    Click the accept button in a JupyterLab dialog (if one is open).
-    ${accept} =    Get WebElements    css:${JLAB CSS ACCEPT}
-    Run Keyword If    ${accept}    Click Element    ${accept[0]}
+    [Documentation]    Click the last button in a JupyterLab dialog (if one is open).
+    # Closing all tabs at once might create a pop-up for each tab. Let's get the
```

Comment:
This change seems to radically alter the meaning of this keyword, as prompts can come into existence for all kinds of reasons, not just closing tabs.

The looping behavior could be done with another keyword, e.g. `Accept all JupyterLab Prompts`, which could reuse the status-quo keyword until it didn't return anything.

Both should probably return the _number_ of prompts that were closed.

Reply:
Done in latest commit